src/Exception/EntityNotFoundException.php

```diff
@@ -0,0 +1,20 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Linio\Common\Exception;
+
+use Throwable;
+
+class EntityNotFoundException extends ClientException
+{
+    public function __construct(
+        string $token = ExceptionTokens::ENTITY_NOT_FOUND,
+        string $message = ExceptionTokens::ENTITY_NOT_FOUND,
```

Comment:
`__toString` is definitely cleaner. But if you need more info in the message, then you need to inject stuff into the exception. I prefer message as the first arg to keep things simple.